muffnn/mlp/mlp_classifier.py

```diff
@@ -262,3 +262,8 @@ def __getstate__(self):
             state['n_classes_'] = self.n_classes_
 
         return state
+
+    def score(self, X, y):
+        accuracy = np.array(y) == self.predict(X)
+        accuracy = accuracy[np.array(y) != -1].mean(axis=0).mean()
```

Comment:
Right now, the loss function takes a mean of errors without regard to variable:

`        if self.multilabel_:
            cross_entropy = tf.nn.sigmoid_cross_entropy_with_logits(
                t, tf.cast(self.input_targets_, np.float32))
`
`        self._obj_func = tf.reduce_mean(cross_entropy)
`
If we were to change it, I think we want to make the columns of response equal in weight and so this score function should do that. What do you think?